### PR TITLE
Implemented testing

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
test1
test2
test3
test4
test5
test6

Testing of movie widgets.

### CODE REVIEW

File: test/build_widget.dart [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Widget buildWidget(Widget child) {
+  return MaterialApp(
+    debugShowCheckedModeBanner: false,
+    title: 'Flutter Demo',
+    theme: ThemeData(
+      primarySwatch: Colors.blue,
+    ),
+    home: child,
+  );
+}
```
Comment (FranciscoGaraventa) [inline test/build_widget.dart]: You missed a newline at end of file

**Review by MarcosCGlobant — APPROVED** [review]
LGTM, just fix Fran's comment

File: test/build_widget.dart (hunk 1 above)
Comment (LeonardoMolleker) [reply test/build_widget.dart]: done

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**